roles/task-shortcuts/files/uug_ansible_wrapper.py

```diff
@@ -230,27 +230,39 @@ def show_about_dialog(self, _):
         about_dialog.show()
 
     def sub_command_exited(self, _, exit_status):
-        """Displays a dialog informing the user whether the gksudo and
+        """Displays a dialog informing the user whether the pkexec and
            ansible-pull commands completely successfully or not"""
         for checkbox in self.checkboxes:
             checkbox.set_sensitive(True)
         self.cancel_button.set_sensitive(True)
         self.run_button.set_sensitive(True)
-
         if exit_status == 0:
             success_msg = "Your machine has been configured for: %s" \
                           % (",".join(TAGS))
             show_dialog(self, Gtk.MessageType.INFO, Gtk.ButtonsType.OK,
                         "Complete", success_msg)
             logging.info("ansible-pull succeeded")
-        # 65280 should be the exit code if the gksudo dialog is dismissed
-        elif exit_status == 65280:
-            gksudo_err_msg = "Either an incorrect password was entered or" \
-                             " the password dialog was closed." \
-                             " Please try again"
+        # 126 should be the exit code if the pkexec dialog is dismissed and
+        # 127 is the exit code if authentication fails. All other exit codes
+        # come from the called application
+        elif exit_status == 126:
```

Comment:
Currently, this stuff isn't working. It seems like `pkexec` when run on its own does exit with the correct codes (126 and 127); however, this keeps receiving the values 32226 and 32512 respectively under the described conditions.